ts/demoTodo.ts

```diff
@@ -0,0 +1,17 @@
+type TypeTodo = [
```

Comment:
`TypeTodo` is not correct naming. Types are **entities** in your business logic. Your application is about todo. So this type is just `Todo`. No need to place `Type` or other information to the naming